blazor/accumulation-chart/legend.md

```diff
@@ -293,6 +293,65 @@ When the legend items exceed legend bounds, paging will be enabled by default. E
 
 ![Paging for Legend in Blazor Accumulation Chart](images/legend/blazor-accumulation-chart-legend-paging.png)
 
+### Paging Customization
+
+The legend paging arrow elements can be customized by using the `ArrowSize` property in the `AccumulationChartLegendPageSettings`, and the page numbers can be customized using `AccumulationChartLegendPageSettingsTextStyle`.
```

Comment:
Add API link for below.
`ArrowSize`
`AccumulationChartLegendPageSettings`
`AccumulationChartLegendPageSettingsTextStyle`